.idea/runConfigurations/ch_gov_uk_on_chs_dev.xml

```diff
@@ -0,0 +1,19 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
We can. However, I added it facilitate debugging of `ch.gov.uk`. It can be seen as forming a pair with `start_debug.sh`. Should I remove both of these files from source code control?

Reply:
Lets keep them in for now then, at least until we're ready to go develop.

Reply:
:+1